modules/ROOT/pages/customize.adoc

```diff
@@ -0,0 +1,107 @@
+
+* https://help.salesforce.com/articleView?id=rss_headline.htm&type=5[Headline, role=external, window=blank]
+* https://help.salesforce.com/articleView?id=rss_setup_nav_menu.htm&type=5[Navigation Menu, role=external, window=blank]: You can update *Max Items to Display (2-15)*,
+*Menu Item Alignment*, and use the *Navigation Menu* to display, add, remove, and customize options. *Home*
+is always present and is always the first item. The following properties can be customized:
+** *Name*
+** *Type*
+** *Page*
+** *Publicly available*
+* https://help.salesforce.com/articleView?id=rss_cms_collection.htm&type=5[CMS Collection, role=external, window=blank]
+* *API Carousel*: Displays APIs and API versions that are available to the community from Anypoint Exchange.
+* https://help.salesforce.com/articleView?id=community_builder_rich_text_component.htm&type=5[Rich text editor, role=external, window=blank]
+* https://help.salesforce.com/articleView?id=rss_html_editor.htm&type=5[HTML editor, role=external, window=blank]
+
+Components that are typically used in *API details* pages include:
+
+* *API Header*: Specifies the API name and description.
+* *Tabs*: Contains the following components: *API Documentation Viewer*, which displays API Exchange information,
+*API Console*, which renders information from the API RAML specification, and *Feed*, which displays comments,
+discussions, and questions.
+* *API Access Requestor*: Registers a client application and requests access to the API.
+
+Components that are typically used in *My Application* pages include:
+
+* *Application Listing*: Lists all client applications that the member has registered and can access.
+
+To reorder items on a page, simply drag and drop them.
+
+== Theme
+
+Select the *Theme* paintbrush icon in the top left navigation to access the following general branding parameters:
+
+* *Colors*: These settings include the site's *General* color scheme used throughout the community,
+including paragraph, heading, and button text.
+* *Images*: Specify the site's *Company Logo* and the login page's *Background Image*.
+* *Fonts*: Specify the site's fonts, which include a *Primary Font* and *Header Fonts*. You can also set the
+site's *Text Case*.
+* https://help.salesforce.com/articleView?id=community_builder_theme_adjust.htm&type=5[Theme Settings, role=external, window=blank]
+
+You can also edit CSS rules to adjust your community styling by selecting the *Theme* drop-down list arrow
+located next to the *Theme* title and selecting *Edit CSS*.
+
+Refer to https://help.salesforce.com/articleView?id=community_builder_theme_customize.htm&type=5[Customize Your Community Theme, role=external, window=blank].
+
+== Pages
+
+Select the *Pages* icon, which resembles a bulleted list, in the top left navigation to display the name of the current
+page or select another page in the community to view and edit. The *Pages* menu also offers a *New Page* option.
+
```

Comment:
Should we be documenting creating new pages here, in a text that is focused on customizing existing pages? 
Consider keeping the mention of the New Page option, but documenting new pages elsewhere...unless we want to change this topic to creating and customizing....

Reply:
This is a good point. This topic describes ACM's Builder UI, which is used for customizing the community, often in ways that involve adding things, such as adding links to the Navigation Menu component, adding components to a page, and adding pages to the community. The text isn't limited to customizing existing pages, and the current description creates that false impression and needs to be changed. I edited the topic introduction to emphasize this.

Also, checking the latest UI shows that the page does not accurately describe the latest interface. In the current version there are in fact a **Pages** list, which includes adding new pages, and an icon for **Page Structure**, which is just for selecting and removing components on the page. Rewrote to fix this error.